apps/backoffice/src/app/api/services/[serviceId]/route.ts

```diff
@@ -127,7 +122,7 @@ export const PATCH = withJWTAuthHandler(
         `An Error has occurred while creating service: userId=${backofficeUser.id} , institutionId=${backofficeUser.institution.id} , serviceId=${params.serviceId}`,
```

Comment:
Is this log correct?

Reply:
no, it's a typo.

Reply:
fixed with 6fb6ad1ded4b72bcc583c7421e3d6bf7f54fe9e4